src/components/Icons/Apple.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const AppleIcon = ({ size, color, children, ...props }) => (
```

Comment:
I don't think we should provide the ability to customize colors for corporate logos (Apple and LinkedIn in this PR), as companies are sensitive to depiction of their brand.